src/app/screen/input-housework/input-housework.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="home-container">
+  <div class="container">
+    <span>家事を入力してね</span>
+    <div class="underline"></div>
+    <div class="input-name"></div>
+    <div class="input-frequency"></div>
+    <div class="circles">
+      <div><img src="assets/images/circle.svg" alt="menu" class="button button-1"></div>
+      <div><img src="assets/images/circle.svg" alt="menu" class="button button-25"></div>
+      <div><img src="assets/images/circle.svg" alt="menu" class="button button-50"></div>
+      <div><img src="assets/images/circle.svg" alt="menu" class="button button-75"></div>
+      <div><img src="assets/images/circle.svg" alt="menu" class="button button-100"></div>
+    </div>
+    <div class="page">
+      <img src="assets/images/button.svg" alt="menu" class="back">
```

Comment:
タグ内の属性が多いときは改行を入れましょう。
alt属性のテキストを適切に指定しましょう。
```suggestion
      <img
        src="assets/images/button.svg"
        alt="back button"
        class="back"
     >
```